### ISSUE TITLE
Need better names for Initial load of Change and Create Changeable

### ISSUE THREAD

**Issue by fosterlynn** [valnet/valuenetwork#406]
Those pattern names are not very clear.  One suggestion: Start Workflow, Continue Workflow.

To make it more generic, I tried Create Workflow Resource and Change Workflow Resource.  These don't save.  Probably too long.

**Comment by bhaugen** [comment]
>
> To make it more generic, I tried Create Workflow Resource and Change
> Workflow Resource. These don't save. Probably too long.
>
>
No, the length was ok. I just changed them to your new names, they saved
fine. Don't know what was wrong before...

**Comment by TiberiusB** [comment]
Do we have this updated in SENSORICA's NRP? Some people got confused with these names, I received feedback.

**Comment by bhaugen** [comment]
Nope. Still the same way. People need to engage with the problem and help figure out solutions.

**Comment by gcassel** [comment]
^ agreed; I think anonymous complaints should either be made more direct, or be marginalized

**Comment by bhaugen** [comment]
I do mostly ignore anonymous complaints and also reports of confusion, not user friendly, not intuitive, etc, because I can't do anything about them. Which usually frustrated Tibi, but it's just impossible to focus anywhere.
When people engage with the design problem it becomes useful: in this case, what is happening there? The same resource is both and input to and an output from the same process. What do we call such an input? What do we call such an output? How do we signify that it is, in fact, the same thing going in and out, just with some changes?
Here's a design document where we went around and around (and around) this problem: https://docs.google.com/document/d/1SpJuFv9JWzMcSQ_AvjBgiwCaK8G6pz6rqtN-C9iGmS8/edit?usp=sharing
That's if you think this stuff is easy.
